.gitignore

```diff
@@ -1,3 +1,4 @@
+*.core
```

Comment:
Should we really be ignoring core files? If we are generating cores, we should investigate why and fix.

Reply:
Thanks for pointing to this. I get a lot of core dumps whenever I use visit on LC. I assumed it was because I typically exit visit with a ctrl+c from the terminal, but it sounds like there are more serious reasons?

Reply:
I'm not sure about ctrl-c. I am concerned, however, that adding cores to .gitignore will let them fall off the radar.  IIRC, we started getting cores on exit with vtk-8, but no one has ever had the cycles to investigate.

Reply:
I see. I've removed these from .gitignore.